clients/src/main/java/org/apache/kafka/common/config/TopicConfig.java

```diff
@@ -93,6 +93,11 @@ public class TopicConfig {
             "deletes the old segments. Default value is -2, it represents `retention.bytes` value to be used. The effective value should always be " +
             "less than or equal to `retention.bytes` value.";
 
+    public static final String REMOTE_LOG_DISABLE_POLICY_CONFIG = "remote.log.disable.policy";
+
+    public static final String REMOTE_LOG_DISABLE_POLICY_DOC = "Determines whether tiered data for a topic should be retained or " +
```

Comment:
Can you add a note about the valid values in the doc and briefly talk about the respective behavior of both values?